Formula/jpeg-xl.rb

```diff
@@ -0,0 +1,57 @@
+class JpegXl < Formula
+  desc "New file format for still image compression"
+  homepage "https://jpeg.org/jpegxl/index.html"
+  url "https://gitlab.com/wg1/jpeg-xl/-/archive/v0.2/jpeg-xl-v0.2.tar.bz2"
+  sha256 "f0933c796f95ee905efa7a677367c0d57678b9587c2e967ea30d72e9405cca72"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "brotli"
+  depends_on "giflib"
+  depends_on "ilmbase"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "openexr"
+  depends_on "webp"
+
+  # These resources are versioned according to the script supplied with jpeg-xl to download the dependencies:
+  # https://gitlab.com/wg1/jpeg-xl/-/blob/master/deps.sh
```

Comment:
```suggestion
  # https://gitlab.com/wg1/jpeg-xl/-/blob/v#{version}/deps.sh
```
Lets refer to dependencies for the current version, not the one in master.

Reply:
Just this one and the spaces in between the `resource` blocks and we should be good here, I think.